src/utils/issue.ts

```diff
@@ -1,26 +1,32 @@
 import { Context } from "../types/context";
-import { Issue, ISSUE_TYPE, PullRequest, Review } from "../types/payload";
+import { GitHubIssueSearch, Review } from "../types/payload";
 import { getLinkedPullRequests, GetLinkedResults } from "./get-linked-prs";
 
 export function isParentIssue(body: string) {
   const parentPattern = /-\s+\[( |x)\]\s+#\d+/;
   return body.match(parentPattern);
 }
 
-export async function getAssignedIssues(context: Context, username: string): Promise<Issue[]> {
+export async function getAssignedIssues(context: Context, username: string): Promise<GitHubIssueSearch["items"]> {
   const payload = context.payload;
 
   try {
-    return await context.octokit.paginate(
-      context.octokit.issues.listForRepo,
-      {
-        owner: payload.repository.owner.login,
-        repo: payload.repository.name,
-        state: ISSUE_TYPE.OPEN,
+    return await context.octokit
+      .paginate(context.octokit.search.issuesAndPullRequests, {
+        q: `org:${payload.repository.owner.login} assignee:${username} is:open`,
```

Comment:
`is:issue` in this query for issues as we were asked by gentlementlegen

Reply:
When I add `is:issue` then there's no need to `!issue.pull_request &&` right? Since the query already filters for issues only.